web/b3desk/endpoints/join.py

```diff
@@ -273,19 +274,19 @@ def join_waiting_meeting_from_sip(visio_code):
 @bp.route("/meeting/visio_code", methods=["POST"])
 @check_oidc_connection(auth)
 def visio_code_connexion():
-    # csrf
-    # captcha?
     visio_code = request.form.get("visio_code")
     meeting = get_meeting_by_visio_code(visio_code)
     if not meeting:
         flash("Le code de connexion saisi est erroné", "error")
+        visio_code_attempt_counter_update(False)
         return redirect(url_for("public.home"))
+    visio_code_attempt_counter_update(True)
     return join_waiting_meeting_with_visio_code(meeting)
 
 
 def join_waiting_meeting_with_visio_code(meeting):
     meeting_fake_id = str(meeting.id)
-    creator = db.session.get(User, meeting.user_id)
```

Comment:
Please use `db.session.get(User, ...)` instead of `User.query.get(...)`. Otherwise this raise SQLAlchemy warnings:

```
tests   | tests/meeting/test_captcha.py::test_join_meeting_with_wrong_visio_code_until_captcha
tests   |   /opt/bbb-visio/b3desk/endpoints/join.py:289: LegacyAPIWarning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
tests   |     creator = User.query.get(meeting.user_id)
```